src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -10463,9 +10478,17 @@ namespace HVACUnitarySystem {
 						CompOffFlowRatio = UnitarySystem( UnitarySysNum ).MSCoolingSpeedRatio( CoolSpeedNum - 1 );
 					}
 				} else {
-					if ( CoolSpeedNum <= 1 ) {
+					if ( CoolSpeedNum < 1 ) {
```

Comment:
Separating out speed = 0, 1, and >1 so that there is flexibility in case variable speed coils need something special (different author, different method).